install/installer/pkg/components/spicedb/migrations.go

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License.AGPL.txt in the project root for license information.
+
+package spicedb
+
+import (
+	"fmt"
+
+	"github.com/gitpod-io/gitpod/installer/pkg/cluster"
+	"github.com/gitpod-io/gitpod/installer/pkg/common"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/utils/pointer"
+)
+
+func migrations(ctx *common.RenderContext) ([]runtime.Object, error) {
+
+	cfg := getExperimentalSpiceDBConfig(ctx)
+	if cfg == nil {
+		return nil, nil
+	}
+
+	if cfg.DisableMigrations {
```

Comment:
out of curiosity: in which cases would you want to disable migrations?

Reply:
We do this for the CloudSQL in SaaS case where we manage the migrations (across regions) ourselves. Mostly wanted to have the option in case we need it